packages/browser/src/extensions/replay/external/lazy-loaded-session-recorder.ts

```diff
@@ -646,8 +648,6 @@ export class LazyLoadedSessionRecording implements LazyLoadedSessionRecordingInt
             this._endpoint = config?.endpoint
         }
 
-        this._setupSampling()
```

Comment:
all setupSampling did was register the listener to make a sampling decision on session id change, which is now handled in the one session id callback